azkaban-common/src/main/java/azkaban/project/ProjectManager.java

```diff
@@ -268,16 +232,10 @@ public Project createProject(String projectName, String description,
           "Project names must start with a letter, followed by any number of letters, digits, '-' or '_'.");
     }
 
-    if (projectsByName.containsKey(projectName)) {
```

Comment:
If the project already exists, a DB exception will be thrown now. Right? How is that new exception handled differently from the existing one?